DigitalOffice/Tests/AuthenticationServiceTests/Commands/UserLoginCommandTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using FluentValidation;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using Moq;
+using NUnit.Framework;
+using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.AuthenticationService.Token.Interfaces;
+using LT.DigitalOffice.AuthenticationService.Commands;
+using LT.DigitalOffice.AuthenticationService.Models;
+using LT.DigitalOffice.AuthenticationService.Token;
+using LT.DigitalOffice.AuthenticationService.Validators;
+using LT.DigitalOffice.AuthenticationService.Commands.Interfaces;
+using LT.DigitalOffice.AuthenticationServiceUnitTests.UnitTestLibrary;
+using LT.DigitalOffice.AuthenticationService.Broker.Senders.Interface;
+using LT.DigitalOffice.AuthenticationServiceTests.Broker.Senders;
+
+namespace LT.DigitalOffice.AuthenticationServiceUnitTests.Commands
+{
+    public class UserLoginCommandTests
+    {
+        private IValidator<UserLoginCredentialsRequest> validator;
+        private UserLoginCredentialsRequest newUserCredentials;
+        private UserCredentialsResponse brokerResponse;
+        private Mock<ISender<string, IUserCredentialsResponse>> brokerSender;
+        private IUserLoginCommand command;
+        private NewToken newToken;
+        private string userEmail;
```

Comment:
Why didnt you just create mocks for ISender, NewToken and IValidator?

Reply:
ISender already mocking and due to the peculiarities of creating a jwt token in NewToken, its mocking will not work

Reply:
I don't understand why the NewToken mock doesn't work. (In my opinion, no problem)
But what about validator mock?

Reply:
Fixed